src/Tribe/Tickets_Handler.php

```diff
@@ -124,6 +124,47 @@ public function print_checkedin_totals() {
 	}
 
 	/**
+	 * Get the total capacity event.
+	 *
+	 * @param $event_id int (null)
+	 *
+	 * @since TBD
+	 *
+	 * @return int number of tickets ( -1 means unlimited )
+	 */
+	public function get_total_event_capacity( $event_id = null ) {
+		if ( null === $event_id ) {
+			global $post;
+			$event_id = $post->ID;
```

Comment:
@bordoni Since we only use the ID here, is there any reason to not use `Tribe__Main::post_id_helper( $post )` ?